packages/config/src/validation.ts

```diff
@@ -120,20 +120,88 @@ export const isValidBrowserList = (_key: string, browsers: any): ErrResult | tru
+
+      if (!isValidStopIfAnyPasses) {
+        return false
+      }
+    }
+
+    // if the strategy is 'detect-flake-and-pass-on-threshold', passesRequired must be provided and must be an integer greater than 0
+    if (strategy === 'detect-flake-and-pass-on-threshold') {
+      if (options.stopIfAnyPassed !== undefined) {
+        return false
+      }
+
+      const isValidPassesRequired = _.isInteger(options.passesRequired) && options.passesRequired > 0 && options.passesRequired <= options.maxRetries
+
+      if (!isValidPassesRequired) {
+        return false
+      }
+    }
+  }
+
+  return true
+}
+
 export const isValidRetriesConfig = (key: string, value: any): ErrResult | true => {
   const optionalKeys = ['runMode', 'openMode']
+  const experimentalOptions = ['experimentalStrategy', 'experimentalOptions']
+  const experimentalStrategyOptions = ['detect-flake-but-always-fail', 'detect-flake-and-pass-on-threshold']
+
   const isValidRetryValue = (val: any) => _.isNull(val) || (Number.isInteger(val) && val >= 0)
   const optionalKeysAreValid = (val: any, k: string) => optionalKeys.includes(k) && isValidRetryValue(val)
 
   if (isValidRetryValue(value)) {
     return true
   }
 
-  if (_.isObject(value) && _.every(value, optionalKeysAreValid)) {
-    return true
+  if (_.isObject(value)) {
+    const traditionalConfigOptions = _.omit(value, experimentalOptions)
+    const experimentalConfigOptions = _.pick<any>(value, experimentalOptions)
+
+    const isTraditionalConfigValid = _.every(traditionalConfigOptions, optionalKeysAreValid)
+
+    // if optionalKeys are only present and are valid, return true.
+    // The defaults for 'experimentalStrategy' and 'experimentalOptions' are undefined, but the keys exist, so we need to check for this
+    if (isTraditionalConfigValid && !Object.keys(experimentalConfigOptions).filter((key) => experimentalConfigOptions[key] !== undefined).length) {
+      return true
+    }
+
+    // check experimental configuration. experimentalStrategy MUST be present if experimental config is provided and set to one of the provided enumerations
+    if (experimentalConfigOptions.experimentalStrategy) {
+      // make sure the strategy provided is one of our valid enums
+      const isValidStrategy = experimentalStrategyOptions.includes(experimentalConfigOptions.experimentalStrategy)
+
+      // if a strategy is provided, and traditional options are also provided, such as runMode and openMode, then these values need to be booleans
+      const openAndRunModeConfigOptions = _.pick(value, optionalKeys)
+      const isValidRunAndOpenModeConfigWithStrategy = _.every(openAndRunModeConfigOptions, _.isBoolean)
+
+      // if options aren't present (either undefined or null) or are configured correctly, return true
+      if (isValidStrategy && isValidRunAndOpenModeConfigWithStrategy && (
+        experimentalConfigOptions.experimentalOptions == null ||
```

Comment:
I think we should bounce on `null` here since it isn't a valid option. its either `undefined` or what is expected for the given `strategy`.
```suggestion
        experimentalConfigOptions.experimentalOptions === undefined ||
```